db/migrate/20241014210800_create_evaluator_invitations.rb

```diff
@@ -0,0 +1,16 @@
+class CreateEvaluatorInvitations < ActiveRecord::Migration[7.2]
+  def change
+    create_table :evaluator_invitations do |t|
+      t.references :challenge, null: false, foreign_key: true
+      t.references :phase, null: false, foreign_key: true
+      t.string :first_name, null: false
+      t.string :last_name, null: false
+      t.string :email, null: false
+      t.datetime :last_invite_sent, null: false
```

Comment:
I was wrong, `last_invite_sent` is nullable. we can't send the invites yet and even when we do they will be a delayed job so the record will be created before the invite is sent and then updated when the job is processed.